src/datastore/firebase.ts

```diff
@@ -91,3 +99,11 @@ export const userRiddleInfoCollection = collection(
 
 export const userRiddleInfoDocument = (id: string) =>
 	doc(db, 'userRiddleInfo', id) as DocumentReference<UserRiddleInfoDb>;
+
+export const usersCollection = collection(
+	db,
+	'users'
+) as CollectionReference<UserDb>;
+
+export const usersDocument = (id: string) =>
+	doc(db, 'users', id) as DocumentReference<UserDb>;
```

Comment:
Z pohledu využití db není pro nás možná úplně efektivní mít pro každého uživatele jeden doc, když pro něj ukládáme jen login, protože to znamená, že při každém fetchování uživatelských možností se musí vrátit záznam z n dokumentů. Možná by byl jednodušší jeden doc s jedním atributem pole = seznam uživatelů. Pro demo účely je to každopádně úplně jedno (stejně nečekáme obrovský provoz :D ).